### DISCUSSION TITLE
Stopping Quarto from deleting pdf image files when rendering

### DISCUSSION THREAD

**Discussion by sapphire008** [quarto-dev/quarto-cli#8297] [Q&A] [answered]
### Description

Hi, I have an asset folder that stores svg and pdf version of the same image. My document references the svg version. When I run `quarto render`, the pdf version of the images are deleted. I have tested that if I convert the image to eps version or other image formats, these other variants do not get deleted by `quarto render`. I have tried with the `--no-clean` flag as well, but these images still got deleted. So my question is how can I keep the pdf version of my images around?

**Comment by cscheid** [comment]
I think that your particular project configuration is confusing our `src/resources/filters/quarto-post/pdf-images.lua` filter. That's likely to be a bug; if you can create a minimal example that reproduces the behavior, we'd appreciate if you could create a new issue. Thanks!

**Reply by sapphire008** [reply, marked as answer]
Issue created here: https://github.com/quarto-dev/quarto-cli/issues/8299
